docs/language/operators/from.md

```diff
@@ -5,11 +5,13 @@
 ### Synopsis
 
 ```
-from <pool>[@<tag>] [ => <leg> ]
+from <pool>[@<branch|id>]
```

Comment:
This `[ => <leg> ]` seemed like copypasta so I dropped it. Please let me know if I'm wrong about this, though!

Also, `@tag` didn't seem to show up in other docs, so I went for the explicit `@<branch|id>` instead. Since the section on "Commitish" in the "Zed command" doc speaks of branch & id, I figured this was ok.